mix.exs

```diff
@@ -29,6 +42,19 @@ defmodule MongoosePush.Mixfile do
   #
   # Type "mix help deps" for more examples and options
   defp deps do
-    []
+    [
+     {:pigeon, git: "https://github.com/rslota/pigeon.git", tag: "6d1e4e3"},
```

Comment:
I can try, but there are some changes that the author may not like. Author of this project had much different plan when comes to pooling.